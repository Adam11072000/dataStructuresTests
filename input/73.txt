init 130
setLabelScore 167 98 0 186
quit