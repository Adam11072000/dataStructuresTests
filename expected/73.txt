init done.
addImage: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
quit done.
